styles.css

```diff
@@ -0,0 +1,7 @@
+.h2 {
+  color: blue;
+}
+
+/* .h2 {
+  color: blue;
+} */
```

Comment:
please remove unused codes instead of commenting it, because it makes your code clean and readable for others